Cargo.toml

```diff
@@ -26,3 +28,4 @@ lazy_static = "1.0"
 [dependencies.onig_sys]
 version = "68.0.1"
 path = "onig_sys"
+default-features = false
```

Comment:
Not sure we need to remove default features here, if we are removing the posix feature from `onig_sys` too.

Reply:
I've fixed that.